metrics.py

```diff
@@ -0,0 +1,115 @@
+"""Metrics to evaluate performances."""
+
+import numpy as np
+from scipy import linalg
+from torch.nn import functional as F
+
+
+def binary_cross_entropy(input, target):
+    """
+    BCE averaged over the voxels of the images.
+    """
+    bce = F.binary_cross_entropy(input, target, reduction='mean')
+    return bce
+
+
+def mse(input, target):
+    """
+    MSE averaged over the voxels of the images.
+    """
+    mse = F.mse(input, target, reduction='mean')
+    return mse
+
+
+def l1_norm(input, target):
+    """
+    L1 norm averaged over the voxels of the images.
+    """
+    l1_norm = np.mean(np.abs(target - input))
```

Comment:
Why are you using both torch and bumpy in these? Shouldn’t this be torch mean and abs?